lib/Convos/Controller/Connection.pm

```diff
@@ -103,6 +109,9 @@ sub update {
     $state = 'reconnect' if $url->to_string ne $connection->url->to_string;
     $connection->url->parse($url);
   }
+  if (my $cmds = $json->{on_connect_commands}) {
```

Comment:
Pretty sure this isn't required. `$user->connection()` above will modify the original object if it exists: https://github.com/Nordaaker/convos/blob/master/lib/Convos/Util.pm#L34